frameworks/digital-outcomes-and-specialists-2/manifests/briefs_search_filters.yml

```diff
@@ -1,10 +1,12 @@
+-
+  name: Status
```

Comment:
Don't think we should be adding this until this ticket has actually been kicked off: https://trello.com/c/LFZSi6r1/105-suppliers-should-be-able-to-filter-by-more-statuses-of-an-opportunity

In particular, the Acceptance Criteria on there talk about having a feature flag. Laura might get rid of that requirement

The thing I'm more worried about is whether people using this interface will properly understand the limitations of that data - this ticket never made it into any sprint :-( https://trello.com/c/ARcnsQLM/133-suppliers-need-to-understand-the-completeness-of-status-data

Reply:
We had a chat about this at standup and decided to stick with just open/closed for now, so I'm going to remove.